gym_duckietown/randomization/config/default.json

```diff
@@ -0,0 +1,11 @@
+{
+    "horz_mode": {
```

Comment:
is it documented somewhere what these numbers mean?

Reply:
i.e. how they are applied in each case? or should one go and look at the code to see how they are applied?

Reply:
I will have a README with a qualitative description, as well as a reference to the line number where each item is applied. I think that should handle it.